### PR TITLE
zh-trans: /docs/contribute/localization.md

### PR DESCRIPTION
**Pull request by liyuan198251**
Signed-off-by: liyuan198251 <li.yuan4@zte.com.cn>

>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Please delete this note before submitting the pull request.
>
> For 1.14 Features: set Milestone to 1.14 and Base Branch to dev-1.14
> 
> For Chinese localization, base branch to release-1.12
>
> For Korean Localization: set Base Branch to dev-1.13-ko.<latest team milestone>
>
> Help editing and submitting pull requests:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> Help choosing which branch to use:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

File: content/zh/docs/contribute/localization.md [hunk 1]
```diff
@@ -0,0 +1,528 @@
...
+## Getting started
+-->
+
+## 入门
+
+<!--
+Localizations must meet some requirements for workflow (*how* to localize) and output (*what* to localize).
+-->
+
+本地化必须满足工作流（*如何*本地化）和输出（本地化*什么*）的一些要求。
```
Comment (Mr-Linus) [inline content/zh/docs/contribute/localization.md]:
```suggestion
本地化必须满足工作流（*如何*本地化）和输出（本地化*内容*）的一些要求。
```

File: content/zh/docs/contribute/localization.md [hunk 2]
```diff
@@ -0,0 +1,528 @@
...
+-->
+
+本地化相关的 [pull request](../create-pull-request) 示例，请参见向 Kubernetes 文档库 [Kubernetes website repo](https://github.com/kubernetes/website) 合入韩语本地化的 [this pull request](https://github.com/kubernetes/website/pull/8636)。
+{{< /note >}}
+
+<!--
+Let Kubernetes SIG Docs know you're interested in creating a localization! Join the [SIG Docs Slack channel](https://kubernetes.slack.com/messages/C1J0BPD2M/). We're happy to help you get started and answer any questions you have.
+-->
+
+让 Kubernetes SIG Docs 知道你对创建本地化感兴趣！加入 [SIG Docs Slack channel](https://kubernetes.slack.com/messages/C1J0BPD2M/)。我们很乐意帮助你起步并回答你的任何问题。
```
Comment (Mr-Linus) [inline content/zh/docs/contribute/localization.md]:
```suggestion
让 Kubernetes SIG Docs 知道你对创建本地化感兴趣！加入 [SIG Docs Slack channel](https://kubernetes.slack.com/messages/C1J0BPD2M/)。我们很乐意帮助你快速上手并回答你的任何问题。
```

File: content/zh/docs/contribute/localization.md [hunk 3]
```diff
@@ -0,0 +1,528 @@
...
+git clone https://github.com/kubernetes/website
+cd website
+```
+
+{{< note >}}
+<!--
+Contributors to `k/website` must [create a fork](https://kubernetes.io/docs/contribute/start/#improve-existing-content) from which to open pull requests. For localizations, we ask additionally that:
+-->
+
+`k/website` 的贡献者必须 [创建一个克隆](https://kubernetes.io/docs/contribute/start/#improve-existing-content)，从创建 PR。对于本地化，我们还要求：
```
Comment (Mr-Linus) [inline content/zh/docs/contribute/localization.md]:
```suggestion
`k/website` 的贡献者必须[创建一个克隆](https://kubernetes.io/docs/contribute/start/#improve-existing-content)用于发起 PR。对于本地化，我们还要求：
```

File: content/zh/docs/contribute/localization.md [hunk 4]
```diff
@@ -0,0 +1,528 @@
...
+-----|-----
+Home | [All heading and subheading URLs](https://kubernetes.io/docs/home/)
+Setup | [All heading and subheading URLs](https://kubernetes.io/docs/setup/)
+Tutorials | [Kubernetes Basics](https://kubernetes.io/docs/tutorials/kubernetes-basics/), [Hello Minikube](https://kubernetes.io/docs/tutorials/stateless-application/hello-minikube/)
+Site strings | [All site strings in a new localized TOML file](https://github.com/kubernetes/website/tree/master/i18n)
+-->
+
+描述 | 网址
+-----|-----
+Home | [All heading and subheading URLs](https://kubernetes.io/docs/home/)
```
Comment (Mr-Linus) [inline content/zh/docs/contribute/localization.md]: 表格内容也可以根据我们的zh导航翻译

File: content/zh/docs/contribute/localization.md [hunk 5]
```diff
@@ -0,0 +1,528 @@
...
+The latest version is **{{< latest-version >}}**, so the most recent release branch is [`{{< release-branch >}}`](https://github.com/kubernetes/website/tree/{{< release-branch >}}).
+-->
+
+最新版本是 **{{< latest-version >}}**，所以最新的发布分支是 [`{{< release-branch >}}`](https://github.com/kubernetes/website/tree/{{< release-branch >}})。
+
+<!--
+### Site strings in i18n/
+-->
+
+### i18n中的网站字符串/
```
Comment (Mr-Linus) [inline content/zh/docs/contribute/localization.md]:
```suggestion
### i18n 中的网站字符串/
```

File: content/zh/docs/contribute/localization.md [hunk 6]
```diff
@@ -0,0 +1,528 @@
...
+### Fork and clone the repo
+-->
+
+### 复制并克隆仓库
+
+<!--
+First, [create your own fork](https://help.github.com/articles/fork-a-repo/) of the [kubernetes/website](https://github.com/kubernetes/website).
+-->
+
+首先，在 [kubernetes/website](https://github.com/kubernetes/website) 中[创建你自己的克隆](https://help.github.com/articles/fork-a-repo/)。
```
Comment (xichengliudui) [inline content/zh/docs/contribute/localization.md]: 创建你自己的克隆，这里的克隆我个人认为：创建你自己的分支，比较合适。

File: content/zh/docs/contribute/localization.md [hunk 7]
```diff
@@ -0,0 +1,528 @@
...
+All localization teams must be self-sustaining with their own resources. We're happy to host your work, but we can't translate it for you.
+-->
+
+所有本地化团队必须使用自身的资源独立工作。我们很高兴支持你的工作，但无法为你翻译。
+
+<!--
+### Fork and clone the repo
+-->
+
+### 复制并克隆仓库
```
Comment (xichengliudui) [inline content/zh/docs/contribute/localization.md]:
```suggestion
### 克隆仓库并创建分支
```

File: content/zh/docs/contribute/localization.md [hunk 8]
```diff
@@ -0,0 +1,528 @@
...
+git clone https://github.com/kubernetes/website
+cd website
+```
+
+{{< note >}}
+<!--
+Contributors to `k/website` must [create a fork](https://kubernetes.io/docs/contribute/start/#improve-existing-content) from which to open pull requests. For localizations, we ask additionally that:
+-->
+
+`k/website` 的贡献者必须[创建一个克隆](https://kubernetes.io/docs/contribute/start/#improve-existing-content)，从创建 PR。对于本地化，我们还要求：
```
Comment (xichengliudui) [inline content/zh/docs/contribute/localization.md]:
```suggestion
`k/website` 的贡献者必须[创建一个分支](https://kubernetes.io/docs/contribute/start/#improve-existing-content)，来打开拉取请求。对于本地化，我们还要求：
```

File: content/zh/docs/contribute/localization.md [hunk 9]
```diff
@@ -0,0 +1,528 @@
...
+
+<!--
+1. Team approvers open development branches directly from https://github.com/kubernetes/website.
+2. Localization contributors work from forks, with branches based on the current development branch.
+-->
+
+1. 团队审批者直接从 https://github.com/kubernetes/website 新建开发分支。
+2. 本地化贡献者创建克隆进行工作，其分支基于当前的开发分支。
```
Comment (xichengliudui) [inline content/zh/docs/contribute/localization.md]:
```suggestion
2. 本地化贡献者创建分支进行工作，其分支基于当前的开发分支。
```

**Comment by xichengliudui** [comment]
对于 `pull request` 我不确定是否需要翻译，`拉取请求`?
@tengqm

**Comment by tengqm** [comment]
> 对于 `pull request` 我不确定是否需要翻译，`拉取请求`?
> @tengqm

可以翻译。我的理念是能翻译就翻译，尽量不要中英文混合不伦不类，除非有特殊理由。

**Pull request merged**